src/steps/installs/index.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
+import { IntegrationConfig } from '../../config';
+import { integrationConfig } from '../../../test/config';
+import { setupWPEngineRecording } from '../../../test/recording';
+import { fetchInstalls } from '.';
+import { fetchSites } from '../sites';
+import { fetchAccounts } from '../accounts';
+
+describe('#fetchSite', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupWPEngineRecording({
+      directory: __dirname,
+      name: 'fetchSite',
+    });
+  });
+
+  afterEach(async () => {
+    await recording.stop();
+  });
+
+  test('should collect data', async () => {
+    const context = createMockStepExecutionContext<IntegrationConfig>({
+      instanceConfig: integrationConfig,
+    });
+
+    await fetchAccounts(context);
+    await fetchSites(context);
+    await fetchInstalls(context);
+
+    expect({
+      numCollectedEntities: context.jobState.collectedEntities.length,
+      numCollectedRelationships: context.jobState.collectedRelationships.length,
+      collectedEntities: context.jobState.collectedEntities,
+      collectedRelationships: context.jobState.collectedRelationships,
+      encounteredTypes: context.jobState.encounteredTypes,
+    }).toMatchSnapshot();
+
+    const sites = context.jobState.collectedEntities.filter((e) =>
+      e._class.includes('Host'),
+    );
+    expect(sites.length).toBeGreaterThan(0);
+    expect(sites).toMatchGraphObjectSchema({
+      _class: ['Host'],
+      schema: {
+        additionalProperties: false,
+        properties: {
+          _rawData: {
+            type: 'array',
+            items: { type: 'object' },
+          },
+          _type: { const: 'wp_engine_site' },
+          name: { type: 'string' },
+        },
+      },
+    });
+
+    const accounts = context.jobState.collectedEntities.filter((e) =>
+      e._class.includes('Account'),
+    );
+    expect(accounts.length).toBeGreaterThan(0);
+    expect(accounts).toMatchGraphObjectSchema({
+      _class: ['Account'],
+      schema: {
+        additionalProperties: false,
+        properties: {
+          _rawData: {
+            type: 'array',
+            items: { type: 'object' },
+          },
+          _type: { const: 'wp_engine_account' },
+          name: { type: 'string' },
+        },
+      },
+    });
+
+    const installs = context.jobState.collectedEntities.filter((e) =>
+      e._class.includes('Application'),
+    );
+    expect(installs.length).toBeGreaterThan(0);
+    expect(installs).toMatchGraphObjectSchema({
+      _class: ['Application'],
+      schema: {
+        additionalProperties: false,
+        properties: {
+          _rawData: {
+            type: 'array',
+            items: { type: 'object' },
+          },
+          _type: { const: 'wp_engine_install' },
+          name: { type: 'string' },
+        },
+      },
+    });
+  });
+});
```

Comment:
Let's add relationships tests below this, here's an example of how to do this here (please update other `index.test.ts` as necessary).

All the relationships collected are stored in `context.jobState.collectedRelationships`.

```
expect(
    context.jobState.collectedRelationships.filter(
      (e) =>
        e._type === Relationships.ACCOUNT_HAS_SITE._type,
    ),
  ).toMatchDirectRelationshipSchema({
    schema: {
      properties: {
        _class: { const: 'HAS' },
        _type: {
          const: 'wp_engine_account_has_site',
        },
      },
    },
  });

  expect(
    context.jobState.collectedRelationships.filter(
      (e) =>
        e._type === Relationships.ACCOUNT_HAS_INSTALL._type,
    ),
  ).toMatchDirectRelationshipSchema({
    schema: {
      properties: {
        _class: { const: 'HAS' },
        _type: {
          const: 'wp_engine_account_has_install',
        },
      },
    },
  });
```